lib/r10k/module/git.rb

```diff
@@ -108,6 +108,8 @@ def parse_options(options)
 
     if @desired_ref == :control_branch && @environment && @environment.respond_to?(:ref)
       @desired_ref = @environment.ref
+    elsif @desired_red == :control_branch && @environment.is_a?(String)
```

Comment:
Typo here

```suggestion
    elsif @desired_ref == :control_branch && @environment.is_a?(String)
```

Reply:
Oops.  Somewhere between testing and removing random debug/pry stuff I must have accidentally mutated an `f` into `d`...